service/chartconfig/v1/resource/chart/mocks.go

```diff
@@ -22,21 +22,23 @@ func (a *apprMock) GetReleaseVersion(customObject v1alpha1.ChartConfig) (string,
 }
```

Comment:
I think I was talking about that already. I find this file name very misleading. Why not adding `_test.go` suffix so we all know it's used only for testing? And using compiler to enforce that.